Resource/doc/FormConfigResolver.md

```diff
@@ -0,0 +1,27 @@
+FormConfigResolver
+==================
+
+Create [`Form`](../../src/Model/Form.php) Model based on action and className. 
+This Resolver uses AnnotationResolver to get all fields from class that are annotated 
+with [`Field`](../../src/Model/Field.php) annotation and have supported Doctrine [`Column`](http://www.doctrine-project.org/api/orm/2.3/class-Doctrine.ORM.Mapping.Column.html) type.
+
+``FormConfigResolver`` provide methods:
+
+ - ``public function resolve(string $action, string $className): Form``
+ 
+    Create [`Form`](../../src/Model/Form.php) model with resolved fields.
+    
+ - ``protected function resolveFields(string $action, string $className): array``
+ 
+    Create Array of fields based on action and class passed. 
```

Comment:
`Creates array` - 3rd person + lowercase.

Lowercase because it is description, not a class name